apis/v1alpha1/gateway_types.go

```diff
@@ -325,7 +313,9 @@ type GatewayTLSConfig struct {
 	// string for both the group and the resource, the resource defaults to "secrets".
 	// An implementation may support other resources (for example, resource
 	// "mycertificates" in group "networking.acme.io").
+	//
 	// Support: Core (Kubernetes Secrets)
+	//
```

Comment:
What do these added line breaks do?

Reply:
<img width="1053" alt="Screen Shot 2021-01-20 at 3 54 59 pm" src="https://user-images.githubusercontent.com/9917/105128809-dc29fc00-5b37-11eb-8aa5-85f57a1c00e0.png">

Fixes this run-on paragraph.